Source/commands/for_CommandCustomAttribute/when_command_coordinator_throws_exception_with_error_callback.ts

```diff
@@ -26,5 +26,5 @@ describe('when command coordinator throws exception with error callback', () =>
           }, 100);
     });
 
-    it('should call error callback with exception', () => (context.attribute.error as sinon.SinonStub).calledWith(exception).should.be.true);
+    it('should call error callback with exception', () => (context.attribute.error as sinon.SinonStub).calledWith({error: exception}).should.be.true);
```

Comment:
Shouldn't the setup for the handle command in this spec be throwing an exception, not returning a rejected / exceptioned promise? Otherwise it's identical to `when_command_coordinator_promise_throws_exception_with_error_callback.ts`.
```ts
        context.commandCoordinator.handle = (command: ICommand) => {
            throw exception;
        };
```